vnext/Scripts/JustMyXaml.ps1

```diff
@@ -0,0 +1,15 @@
+# This script enables or disables VS 2019's JustMyXaml feature
+[CmdletBinding()]
+param([bool]$Enable)
+$instanceId = & "C:\Program Files (x86)\Microsoft Visual Studio\Installer\vswhere.exe" -property instanceId
+$hiveFile = "$($env:LocalAppData)\Microsoft\VisualStudio\16.0_$instanceId\privateregistry.bin"
+& reg.exe load HKLM\VS2019_HIVE $hiveFile | Out-Null
+New-PSDrive -Name VS2019 -PSProvider Registry -Root HKLM\VS2019_HIVE  -ErrorAction Stop | Out-Null
+$currentValue = (Get-ItemProperty VS2019:\Software\Microsoft\VisualStudio\16.0_9f8d4bd3\Debugger -Name EnableXamlVisualDiagnosticsJustMyXaml).EnableXamlVisualDiagnosticsJustMyXaml
```

Comment:
>0_9f8d4bd3 [](start = 77, length = 10)

Should we use the $instanceId here and on the line 12?

Reply:
yes good catch